topology/Portland State University/PDX-OIT/PDX-Coeus.yaml

```
@@ -0,0 +1,154 @@
+      # relationship with the OSG (e.g., principal investigators,
+      # local administrator contact for OSG Hosted CEs)
+      # Site Contact:
+        # Primary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+        # Secondary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+        # Tertiary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+
+    # FQDN is the fully qualified domain name of the host running this resource
+    FQDN: pdx-coeus-ce1.svc.opensciencegrid.org
+    ### FQDNAliases (optional) are any other DNS aliases by which this host can be accessed
+    # FQDNAliases:
+    #   - <HOSTNAME1>
+    #   - <HOSTNAME2>
+
+    ### DN (optional except for XCache resources) is the DN of the host cert of the resource
+    # in OpenSSL 1.0 format (i.e. /DC=org/DC=incommon/C=US/...)
+    # DN: <DN>
+
+    # Services is one or more services provided by this resource;
+    # valid services are listed in topology/services.yaml with the format "<SERVICE NAME>: <ID>"
+    Services:
+      CE:
+        # Description is a brief description of the service
+        Description: Hosted CE
+        ### Details (optional)
+        # Details:
+        #   # hidden
+        #   hidden: false
+        #   ### uri_override (optional, use if your service is on some non-standard URL)
+        #   # uri_override: <HOST>:<PORT>
+        #   ### sam_uri (optional)
+        #   # sam_uri: htcondor://...
+        #   ### endpoint (for perfSONAR services)
+        #   # endpoint: <HOSTNAME>
+
+      # Other services if you have any
+      # <SERVICE NAME>:
+      # ...
+
+    ### Tags (optional) is a list of tags associated with the resource.
+    ### Include the tag "CC*" if applicable for a CC* CE.
+    Tags:
+      - CC*
+    #   - <TAG2>
+
+    ### VOOwnership (optional) is the percentage of the resource owned by one or more VOs.
+    ### If part of the resource is not owned by the VO, do not list it.
+    ### The total percentage cannot exceed 100.
+    # VOOwnership:
+    #   <VO1>: <PERCENT>
+    #   <VO2>: <PERCENT>
+
+    ### WLCGInformation (optional) is only for resources that are part of the WLCG
+    # WLCGInformation:
+    #   APELNormalFactor: 0.0
+    #   AccountingName: <name>
+    #   HEPSPEC: 0
+    #   InteropAccounting: true
+    #   InteropBDII: true
+    #   InteropMonitoring: true
+    #   KSI2KMax: 0
+    #   KSI2KMin: 0
+    #   StorageCapacityMax: 0
+    #   StorageCapacityMin: 0
+    #   TapeCapacity: 0
+
+  # Other resources if you have any...
+  # <RESOURCE NAME>:
+  # ...
+  PDX-Coeus-Squid:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Jeffrey Michael Dost
+          ID: 3a8eb6436a8b78ca50f7e93bb2a4d1f0141212ba
+      Security Contact:
+        Primary:
+          Name: Jeffrey Michael Dost
+          ID: 3a8eb6436a8b78ca50f7e93bb2a4d1f0141212ba
```

```suggestion
        Primary:
          Name: Marko Markoc
          ID: OSG1000189
        Secondary:
          Name: Gary Sandine
          ID: OSG1000190
        Tertiary:
          Name: Jim Stapleton
          ID: OSG1000191
      Security Contact:
        Primary:
          Name: Marko Markoc
          ID: OSG1000189
        Secondary:
          Name: Gary Sandine
          ID: OSG1000190
        Tertiary:
          Name: Jim Stapleton
          ID: OSG1000191
```

The site admins are responsible for the Squid service